tendermint/src/lite/verifier.rs

```diff
@@ -15,30 +15,39 @@ use crate::lite::error::{Error, Kind};
 use crate::lite::{
     Commit, Header, Height, Requester, SignedHeader, TrustThreshold, TrustedState, ValidatorSet,
 };
+use anomaly::ensure;
+use std::ops::Add;
 
 /// Returns an error if the header has expired according to the given
 /// trusting_period and current time. If so, the verifier must be reset subjectively.
 fn is_within_trust_period<H>(
     last_header: &H,
-    trusting_period: &Duration,
-    now: &SystemTime,
+    trusting_period: Duration,
+    now: SystemTime,
```

Comment:
Both `SytemTime` and `Duration` implement the `Copy` trait. Hence, these references where not needed.